### PR TITLE
[Cache] Document `cache:pool:clear`’s new `--exclude` option

### PR DESCRIPTION
**Pull request by MatTheCat**
Feature PR
* https://github.com/symfony/symfony/pull/51058

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

File: cache.rst [hunk 1]
```diff
@@ -733,6 +733,16 @@ Clear all cache pools:
 
     The ``--all`` option was introduced in Symfony 6.3.
 
+Clear all cache pools except some:
+
+.. code-block:: terminal
+
+    $ php bin/console cache:pool:clear --all --exclude=my_cache_pool
```
Comment (OskarStark) [inline cache.rst]: Maybe add another one to showcase, it can be used multiple times ?

Comment (MatTheCat) [reply cache.rst]: Can do :+1: 

Do you think it needs to be another example or can I just do this?

```suggestion
    $ php bin/console cache:pool:clear --all --exclude=my_cache_pool --exclude=another_cache_pool
```

**Review by maxbeckers — APPROVED** [review]

File: cache.rst (hunk 1 above)
Comment (OskarStark) [reply cache.rst]: This is fine 👍

**Review by 94noni — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks for your work on this new feature!

**Pull request merged**